experimental/packages/opentelemetry-sdk-metrics-base/src/InstrumentDescriptor.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import { MetricOptions, ValueType } from '@opentelemetry/api-metrics';
+import { MetricOptions, ValueType } from '@opentelemetry/api-metrics-wip';
```

Comment:
Created a PR to handle these reference updates separately #2638 to avoid cluttering this PR